### ISSUE TITLE
can your model / bifpn be saved in tensorboard?

### ISSUE THREAD

**Issue by XueZ-phd** [MingiJi/FRSKD#6]
To visualize your network architecture more clearly, I intend to save the network architecture including model and bifpn into a tensorboard.

I add the following codes into your [classification/main.py](https://github.com/MingiJi/FRSKD/blob/bed26010383cbf4b8ae3a5d38c414d052c68a6ed/classification/main.py#L97):

`writer = SummaryWriter('./tensorboard/'+args_path)`
 `images, labels = next(iter(train_loader))`
`images = images.cuda()`
`writer.add_graph(model, images)`
`writer.add_graph(bifpn, images)`

However, it returns many erros.

Therefore, I want to find some help from you to visualize your network architecture using a tensorboard.
